docs/blog/2023-04-06-zephyr-3-2.md

```diff
@@ -278,7 +278,7 @@ The following RP2040 powered controllers have board definitions for folks to tes
 
 - Raspberry Pi Pico (`rpi_pico`)
 - SparkFun Pro Micro RP2040 (`sparkfun_pro_micro_rp2040`)
-- Adafruit Keyboar/KB2040 (`adafruit_kb2040`)
+- Adafruit Keyboar/KB2040 (`adafruit_kb2040`) <!-- codespell:ignore keyboar -->
```

Comment:
As far as I can tell, the correct name is not "Keyboard" but "Kee Boar Driver".

I expect that will result in flagging "Kee" though. If so, is it possible to ignore sequences of words such that "Kee" on its own is flagged but not "Kee Boar"?

Reply:
> Kee Boar

Right, i didn't even Google it and assumed it was correctly spelled. It's a weird name either way 🙃 

> "Kee" is flagged but not "Kee Boar"

Good question, on my way to run a couple tests. Worst case scenario, seems like the "identifier" code-wise is `adafruit_kb2040`, so adding a couple of `codespell:ignore` comments on the few places where "kee boar" might appear does not sound like a big issue anyway